server/ec2spotmanager/templates/config/edit.html

```diff
@@ -7,130 +7,7 @@
-      <label><input type="radio" name="ec2_userdata_ff" value="unix"{% if userdata_ff == 'unix' %} checked="checked"{% endif %}> Unix</label>
-      <label><input type="radio" name="ec2_userdata_ff" value="dos"{% if userdata_ff == 'dos' %} checked="checked"{% endif %}> DOS</label>
-      </fieldset>
-      <br/>
-
-      <label for="id_ec2_userdata_macros">EC2 Userdata Macros</label><br/>
-      <input id="id_ec2_userdata_macros" class="form-control" name="ec2_userdata_macros" type="text"
-             value="{{ config.ec2_userdata_macros_dict|dictcsv }}">
-      <input id="id_ec2_userdata_macros_override" name="ec2_userdata_macros_override" type="checkbox"
-             {% if config.ec2_userdata_macros_override %}checked{% endif %} title="Overwrite parent config">
-      <label class="no-bold" for="id_ec2_userdata_macros_override">Overwrite parent config</label>
-      <br/>
-
-      <label for="id_ec2_allowed_regions">EC2 Allowed Regions</label><br/>
-      <input id="id_ec2_allowed_regions" class="form-control" name="ec2_allowed_regions" type="text"
-             value="{{ config.ec2_allowed_regions_list|listcsv }}">
-      <input id="id_ec2_allowed_regions_override" name="ec2_allowed_regions_override" type="checkbox"
-             {% if config.ec2_allowed_regions_override %}checked{% endif %} title="Overwrite parent config">
-      <label class="no-bold" for="id_ec2_allowed_regions_override">Overwrite parent config</label>
-      <br/>
-
-      <label for="id_ec2_max_price">EC2 Maximum Price (per core)</label><br/>
-      <input id="id_ec2_max_price" class="form-control" name="ec2_max_price" type="text"
-             value="{{ config.ec2_max_price|default:"" }}">
-      <br/>
-
-      <label for="id_ec2_tags">EC2 Additional Tags</label><br/>
-      <input id="id_ec2_tags" class="form-control" name="ec2_tags" type="text"
-             value="{{ config.ec2_tags_dict|dictcsv }}">
-      <input id="id_ec2_tags_override" name="ec2_tags_override" type="checkbox"
-             {% if config.ec2_tags_override %}checked{% endif %} title="Overwrite parent config">
-      <label class="no-bold" for="id_ec2_tags_override">Overwrite parent config</label>
-      <br/>
-
-      <label for="id_ec2_raw_config">Additional Raw Configuration</label><br/>
-      <input id="id_ec2_raw_config" class="form-control" name="ec2_raw_config" type="text"
-             value="{{ config.ec2_raw_config_dict|dictcsv }}">
-      <input id="id_ec2_raw_config_override" name="ec2_raw_config_override" type="checkbox"
-             {% if config.ec2_raw_config_override %}checked{% endif %} title="Overwrite parent config">
-      <label class="no-bold" for="id_ec2_raw_config_override">Overwrite parent config</label>
-      <br/>
-
-      <input type="submit" name="submit_save" value="Save" class="btn btn-danger"/>
-      </form>
-    </div>
-  </div>
+  {% if provider == 'ec2' or configuration == configuration.provider.ec2configuration %}
```

Comment:
`configuration.provider.ec2configuration` is old code from the class-based inheritance change. This does nothing anymore.